Importer/Log/Log.php

```diff
@@ -0,0 +1,128 @@
+<?php
+namespace Netdudes\ImporterBundle\Importer\Log;
+
+class Log
+{
+    /**
+     * @var string[]
+     */
+    protected $configurationErrors = [];
+
+    /**
+     * @var string[]
+     */
+    protected $dataErrors = [];
+
+    /**
+     * @var string[]
+     */
+    protected $criticalErrors = [];
+
+    /**
+     * @var string
+     */
+    protected $csvLines;
+
+    /**
+     * @var array
+     */
+    protected $importedEntities;
```

Comment:
As discussed in team, the generic idea of 'Log' class was to gather the errors. Which means that this 'importedEntities' could also not be here. I decided to add this now because I do not think it's completely irrelevant with the purpose of this class. 

If we decide to remove this from here, then this means that in 'u2' we would also have to override the `importData` function of `CsvImporter`.